### PR TITLE
Add Nominating Info

### PR DESCRIPTION
**Pull request by staking4all**
Hi

This PR covers the following
- Pre nominating info
- When to use pools vs stake direct
- How to stake direct on validator
- How to join a nomination pools
- Aimed at users that will stake AVL. So aimed for end users wishing to stake/delegate AVL.

Please assist to add to the menu, was unable to add to the menu. Tried editing _meta.json but didn't stick/work.

Outside of this pr I would recommend moving existing [nomination pools](https://docs.availproject.org/docs/end-user-guide/nomination-pools) to the validator section. The existing write up is how to create a pool, this should be done by validators or providers (wallets, exchanges) looking to pool stake together. Not end users

Hope this helps and contributes to Avail.

Thanks

### CODE REVIEW

**Comment by Genesis3800** [comment]
Hey @staking4all thanks a lot for making this PR. I will go through this in detail and get back 2 u ASAP.

File: pages/docs/end-user-guide/nominating.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+- **Block Nominations**: Block nominations indicates the validator is not looking for more nominations and has therefore blocked any community nominations.
+
+- **Era Points**: Era points are a measure of a validator's contribution to the network over a specific period (era). Validators earn era points based on their performance, including producing and validating blocks, and these points influence their overall reputation and rewards. 
+
+
+### Nominating VS Nomination Pools
+Nominating and nomination pools are both integral parts of the staking process in blockchain networks like Avail. However, they serve slightly different purposes and involve distinct mechanisms:
+
+- **Nominating**: Nominating refers to the process of selecting validators to delegate one's stake to in order to participate in the network's consensus mechanism and earn rewards. When users nominate validators, they essentially entrust them with their tokens to contribute to the security and operation of the network. By nominating validators that perform well and uphold the network's integrity, users can earn staking rewards in return for their contribution.
```
Comment (Genesis3800) [inline pages/docs/end-user-guide/nominating.mdx:58]: I'm pretty sure there's a minimum number of tokens a user needs to have to be an independent nominator.

Can we add that number here?

File: pages/docs/end-user-guide/nominating.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- **Nomination Pools**: On the other hand, nomination pools are collective pools of staked tokens contributed by multiple users who have chosen to nominate a specific validator set. These pools aggregate the stakes of individual nominators and delegate them to the chosen validator. The primary purpose of nomination pools is to increase the nominator's chances of receiving rewards by combining their stakes with others, thereby enhancing the collective weight behind the Pools nomination. In essence, nomination pools allow smaller stakeholders to pool their resources together to compete with larger stakeholders and increase their chances of being selected to validate transactions and earn rewards.
```
Comment (Genesis3800) [inline pages/docs/end-user-guide/nominating.mdx:60]: Do we have a number on how many nominators can be within a single pool?

File: pages/docs/end-user-guide/nominating.mdx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+To determine whether to nominate directly or join a nomination pool, it's essential to evaluate network information available on the [<ins>Staking Targets page</ins>](https://goldberg.avail.tools/#/staking/targets). This information is based on Avail Testnet, this is subject 
+to significant changes on the Avail Mainnet. However does provides valuable insights for making an informed decision. Key parameters to consider include the threshold and minimum nominated values:
+
+- **Threshold**: This represents the minimum AVL token value required for staking and is predetermined as part of the AVL network settings. It can be modified through a governance process. Atbthis moment on Avail Testnet this value is 0. However this is subject to change in Mainnet.
```
Comment (Genesis3800) [inline pages/docs/end-user-guide/nominating.mdx:65]: Spellcheck.

It should be `At this moment`, not `Atbthis moment`

File: pages/docs/end-user-guide/nominating.mdx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+<img src="/img/nominating/nominate-01.png" width="100%" height="100%" />
+
+
+#### 2. Select Wallet & Stake Amount
+You will need select the wallet you wish to nominate from, also the amount. On the amount always leave some AVL for fees, so only nominate 95%-98% of your tokens. Levaing 2%-5% for fees on nominating or any other tranactions you may want to perform.
+Also select where you would like your rewards to acrue to. You have the following options
+
+- **Stash Account (Increase the amount at stake):** This refers to a scenario where the user increases the amount of AVL (Avail token) at stake with their rewards in their stash account. This is like auto-compunding, your rewards are added to your stake and will increase your overall stake for the next era.
+
+- **Stash Account (Do not increase the amount at stake):** In contrast to the previous point, here users choose not to increase the amount of AVL at stake in their stash account. Yur rewards will be sent to this account but ot staked.
```
Comment (Genesis3800) [inline pages/docs/end-user-guide/nominating.mdx:89]: Spellcheck.

It should be `Your`, not `Yur`

File: pages/docs/end-user-guide/nominating.mdx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+
+You will now see your stake in `Waiting`, your stake will become active if you selecd validators that will be elected when the next era starts.
+<img src="/img/nominating/nominate-05.png" width="100%" height="100%" />
+
+
+### How to Join a Nomination pool
+
+#### 1. Navigate to Nomination Pools
+To join a nomination pools navigate to the [<ins>Pools Page</ins>](https://goldberg.avail.tools/#/staking/pools). Select a pool to join where the existing members are less than the max members per pool. Click on `+Join` to enter a pool.
```
Comment (Genesis3800) [inline pages/docs/end-user-guide/nominating.mdx:118]: Same question ser. Do we have a hard number on the max number of members in a pool?

File: pages/docs/end-user-guide/nominating.mdx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+#### 2. Select Wallet & Stake Amount
+You will need select the wallet you wish to nominate from, also the amount. On the amount always leave some AVL for fees, so only nominate 95%-98% of your tokens. Levaing 2%-5% for fees on nominating or any other tranactions you may want to perform.
+Also select where you would like your rewards to acrue to. You have the following options
+
+- **Stash Account (Increase the amount at stake):** This refers to a scenario where the user increases the amount of AVL (Avail token) at stake with their rewards in their stash account. This is like auto-compunding, your rewards are added to your stake and will increase your overall stake for the next era.
+
+- **Stash Account (Do not increase the amount at stake):** In contrast to the previous point, here users choose not to increase the amount of AVL at stake in their stash account. Yur rewards will be sent to this account but ot staked.
+
+- **Controller Account:** The controller account is separate from the stash account and is used to manage the nomination process. This is legacy and will be discontinued going forward.
```
Comment (robin-rrt) [inline pages/docs/end-user-guide/nominating.mdx:91]: @Genesis3800 this probably needs double checking from our side.

Comment (Genesis3800) [reply pages/docs/end-user-guide/nominating.mdx:91]: @robin-rrt pls clarify. I don't understand what exactly needs double checking

**Comment by Genesis3800** [comment]
Hey @staking4all I have pushed a commit taking care of most of the comments. Can you pls take a look at the remaining ones?

File: pages/docs/end-user-guide/nominating.mdx (hunk 1 above)
Comment (staking4all) [reply pages/docs/end-user-guide/nominating.mdx:58]: Hi

Apologies on the delay, been crazy my side.

There is, but to be honest it keeps changing in each testnet. On kate testnet was x, goldberg y, turning z. Might be something we add when we know final config for mainnet

File: pages/docs/end-user-guide/nominating.mdx (hunk 2 above)
Comment (staking4all) [reply pages/docs/end-user-guide/nominating.mdx:60]: At this moment the chain is 200 per pool. I however have recommended the team to increase as it is extremely limiting. Other substrates that use this do not cap at 200

File: pages/docs/end-user-guide/nominating.mdx (hunk 5 above)
Comment (staking4all) [reply pages/docs/end-user-guide/nominating.mdx:118]: At this moment the goldberg is 200 per pool. I however have recommended the team to increase as it is extremely limiting. Other substrates that use this do not cap at 200

This is a setting that can be changed with governance.

File: pages/docs/end-user-guide/nominating.mdx (hunk 6 above)
Comment (staking4all) [reply pages/docs/end-user-guide/nominating.mdx:91]: Controller is legacy and we should remove from docs or stipulate. You will see in new UI for goldberg we do not have option to use controller accounts.

Read here to understand that they will be depecated
https://support.polkadot.network/support/solutions/articles/65000182218#:~:text=Controller%20accounts%20are%20being%20deprecated,account%20using%20the%20Staking%20Dashboard.

Comment (staking4all) [reply pages/docs/end-user-guide/nominating.mdx:91]: Typo

Your rewards will be sent to this account but not staked.

**Pull request closed without merging**